src/pages/UserPage/UserPage.jsx

```diff
@@ -84,7 +86,7 @@ const UserPage = () => {
     setIsEditable(!isEditable);
   }
 
-  // hangles the changes of the inputs
+  // handles the changes of the inputs
   const handleInputChange = (event) => {
     const { name, value } = event.target;
     const matches = name.match(/^(\w+)\[(\d+)\](\[\w+\])?$/);
```

Comment:
This regex could be better in a constant imported as resource in the project.

Reply:
resolved